pywinauto/linux/atspi_element_info.py

```diff
@@ -239,3 +245,32 @@ def document_get_attribute_value(self, attrib):
     def document_get_attributes(self):
         """Return the document's constant attributes"""
         return self.document.get_attributes()
+
+    @property
+    def image(self):
+        """Return AtspiImage interface"""
+        if self.control_type == "Image" or self.control_type == "Icon":
+            image = self.atspi_accessible.get_image(self._handle)
+            return AtspiImage(image)
+        else:
+            raise AttributeError
+
+    def image_get_locale(self):
+        """Return the image locale"""
+        return self.image.get_locale().decode(encoding='UTF-8')
+
+    def image_get_description(self):
+        """Return the image description"""
+        return self.image.get_description().decode(encoding='UTF-8')
+
+    def image_get_position(self):
+        """Return the image position"""
+        return self.image.get_position()
+
+    def image_get_extents(self):
+        """Return the image extents"""
+        return self.image.get_extents()
+
+    def image_get_size(self):
```

Comment:
Do we really need all these image related properties for search class `AtspiElementInfo`? In the future we have plans to use `__slots__` static member for `ElementInfo` derived classes so that memory footprint is minimal.

Reply:
Good point. Will move it directly into atspi_controls.ImageWrapper. Probably even `image` property itself.

Reply:
If we leave `image` property in `AtspiElementInfo` we should think how it looks for end user. I mean something like this: `.child_window(image=AtspiImage(some_pointer))`. Maybe such low level pointer is not a good thing to show publicly. But some text property from image object could be useful for search.

For example `.child_window(image_descr="some text")` might be really useful.

Reply:
The same consideration makes sense for AtspiDocument interface as well. I'd suggest to file a separate issue on this where we can discuss `AtspiElementInfo` interface minimization.

Reply:
Agree. Created issue #806 to continue the discussion.